content/SCALE/DataProtection/CloudSyncTasks.md

```diff
@@ -0,0 +1,108 @@
+
+The remaining options allow tuning the task to your specific requirements.
+
+{{< expand "Specific Options" "v" >}}
+{{< include file="static/includes/TasksCloudSyncAddFields.md.part" markdown="true" >}}
+
+### Scripting and Environment Variables
+
+Advanced users can write scripts that run immediately *before* or *after* the Cloud Sync task. The **Post-script** field is only run when the Cloud Sync task successfully completes. You can pass a variety of task environment variables into the **Pre-** and **Post-** script fields:
+
+* CLOUD_SYNC_ID
+* CLOUD_SYNC_DESCRIPTION
+* CLOUD_SYNC_DIRECTION
+* CLOUD_SYNC_TRANSFER_MODE
+* CLOUD_SYNC_ENCRYPTION
+* CLOUD_SYNC_FILENAME_ENCRYPTION
+* CLOUD_SYNC_ENCRYPTION_PASSWORD
+* CLOUD_SYNC_ENCRYPTION_SALT
+* CLOUD_SYNC_SNAPSHOT
+
+There also are provider-specific variables like CLOUD_SYNC_CLIENT_ID or CLOUD_SYNC_TOKEN or CLOUD_SYNC_CHUNK_SIZE
+
+Remote storage settings:
+* CLOUD_SYNC_BUCKET
+* CLOUD_SYNC_FOLDER
+
+Local storage settings:
+* CLOUD_SYNC_PATH
+
+{{< /expand >}}
+
+### Testing Settings
+
+Test the settings before saving by clicking *Dry Run*. TrueNAS connects to the Cloud Storage Provider and simulates a file transfer. No data is actually sent or received.
+A dialog shows the test status and allows downloading the task logs.  This can also be ran by clicking <i class="material-icons" aria-hidden="true" title="Dry Run">loop</i> after the task has been created.
+
+![DataProtectionCloudSyncAddBoxDryRun](/images/SCALE/DataProtectionCloudSyncDryRun.png "Example: Box Drive Test")
+
+## Cloud Sync Behavior
+
+Saved tasks are activated according to their schedule or by clicking <i class="material-icons" aria-hidden="true" title="Run Now">play_arrow</i>. An in-progress cloud sync must finish before another can begin. Stopping an in-progress task cancels the file transfer and requires starting the file transfer over.
+
+To view logs about a running or the most recent run of a task, click the task *State*.
+
+## Cloud Sync Restore
+
+To quickly create a new Cloud Sync that uses the same options but reverses the data transfer, select <i class="material-icons" aria-hidden="true" title="Restore">history</i> for an existing Cloud Sync on the **Data Protection** page.
+
+![DataProtectionsCloudSyncRestore](/images/SCALE/DataProtectionCloudSyncRestore.png "Cloud Sync Restore")
+
+Enter a new *Description* for this reversed task, define the path to a storage location for the transferred data and click *Restore*.
+
+The restored cloud sync is saved as another entry in **Data protection > Cloud Sync Tasks**.
+
+In case the restore destination dataset is the same as the original source dataset, the restored files might have their ownership altered to *root*. If the original files were not created by *root* and a different owner is required, you can recursively reset ACL Permissions of the restored dataset through the GUI or by running `chown` from the CLI.
```

Comment:
```suggestion
In case the restore destination dataset is the same as the original source dataset, the restored files might have their ownership altered to *root*. If the original files were not created by *root* and a different owner is required, you can recursively reset [ACL Permissions]({{< relref "Permissions.md" >}}) of the restored dataset through the GUI or by running `chown` from the CLI.
```